games-util/springlobby/springlobby-0.264.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+WX_GTK_VER="3.0"
+inherit cmake-utils eutils flag-o-matic wxwidgets
+
+DESCRIPTION="Lobby client for spring rts engine"
+HOMEPAGE="http://springlobby.info"
+SRC_URI="http://www.springlobby.info/tarballs/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+RESTRICT="mirror"
+IUSE="debug +libnotify +nls"
+
+RDEPEND="
+	>=dev-libs/boost-1.35
```

Comment:
Boost needs `:=` since we need to rebuild the package on every major boost upgrade.

Reply:
fixed